docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+
+services:
+
+  ngrok:
+    container_name: skynet_ngrok
+    image: wernight/ngrok
+    ports:
+      - "4040:4040"
+    links:
+      - frontend
+    environment:
+      - NGROK_PORT=app:8100
+      - NGROK_PROTOCOL=http
+      - NGROK_AUTH=3fVv5QDD6Wh5jDUg7CtEg_6tsUV7Nz5gy1M6vZA7myF
```

Comment:
Should this really be exposed? Might be better to have this as an environment variable

Reply:
Very true. Woops. Removed since it'll still work without it. 7bc244ea4d9e701b78e146ba000c377f4fed48ea